app/code/community/Bolt/Boltpay/controllers/OnepageController.php

```diff
@@ -27,69 +27,15 @@ class Bolt_Boltpay_OnepageController
 {
     use Bolt_Boltpay_BoltGlobalTrait;
 
-    public function saveShippingMethodAction() 
```

Comment:
I'm confused. why this file needs to be updated when it doesn't rely on new trait?

Reply:
@daisy1754 Remember, we are doing micro pull request.  There is no perfect way to break down so many interdependent changes.  Trying entire files will have too many changes, so we will focus in on ideas.  The focus here is API Trait capabilities.

This controller/file will rely on api trait capabilities in a later Pull Request.  Specifically, this is the controller that creates the success page that takes a Bolt payload and Bolt signature as parameters.  By default, however, we want it to behave like a standard controller... not necessarily returning JSON and definitely not requiring a signature.

For now, it is just important to understand that it has been enabled to behave with API controller traits.

Reply:
use statement added.